hw/dv/tools/fusesoc.mk

```diff
@@ -12,7 +12,7 @@
 # fusesoc tool and options
 SV_FLIST_GEN_TOOL  ?= fusesoc
 SV_FLIST_GEN_OPTS  += --cores-root ${PROJ_ROOT} --cores-root ${RAL_MODEL_DIR} \
-                      run --target=sim --setup --no-export ${FUSESOC_CORE}
+                      run --build-root=build/hw --target=sim --setup --no-export ${FUSESOC_CORE}
```

Comment:
Please change this to ${BUILD_DIR} instead. Thanks!

Reply:
Actually, there are some other dependencies. Please change this back to the default 'build' (remove the option entirely).